### PR TITLE
Add conventions plugin sample to apply cache fix plugin.

### PR DESCRIPTION
**Pull request by runningcode**
Check the sample here https://github.com/gradle/android-cache-fix-gradle-plugin/tree/no/apply-conventions#applying-the-plugin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,9 +22,36 @@ If you discover an issue related to the Android Gradle Plugin, please file an is
...
+</details>
+<details>
+<summary>Kotlin</summary>
+<br>
+
+```kotlin
+plugins {
+    id("com.android.application") // or "com.android.library"
+  // Add this next line to your existing convention plugin. You can specify the version in the implementation dependency artifact of the convention plugin project's build file.
+    id("com.gradle.android.cache-fix")
```
Comment (kmayoral) [inline README.md:48]: This should be
```
  id("org.gradle.android.cache-fix")
```
instead right?

**Comment by kmayoral** [comment]
Probably also should mention the need to add the dependency entry in `build.gradle.kts` something like this:
```
dependencies {
  ...
  implementation("org.gradle.android.cache-fix:org.gradle.android.cache-fix.gradle.plugin:2.4.6")
  ...
}
```

**Review by ghale — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (runningcode) [reply README.md]: Thanks for pointing this out.

**Comment by runningcode** [comment]
Thanks for the suggestion @kmayoral . I've update the PR with your suggestion.

**Comment by kmayoral** [comment]
Thanks @runningcode, credit to your PR for introducing me to conventions plugins and inspiring me to refactor my project to use them!

File: README.md [hunk 2]
```diff
@@ -42,9 +50,17 @@ plugins {
...
 
 ```kotlin
+// in build.grade.kts for convention plugin build
+dependencies {
+  // ...
+  implementation("org.gradle.android.cache-fix:org.gradle.android.cache-fix.gradle.plugin:2.4.6")
+  // ...
+}
+
+// in com.myconentions.build.gradle.kts
```
Comment (ghale) [inline README.md]: typo

**Pull request merged**